cms/portfolio.php

```diff
@@ -0,0 +1,238 @@
+
+if ($projectTitle3 != NULL && $projectTitleLink3 != NULL && $projectImage3 != NULL && $projectId3!= NULL) {
+    $updatedInfo = updateProject($db, $projectTitle3, $projectTitleLink3, $projectImage3, $projectId3);
+    echo $updatedInfo;
+
+}
+
+$projectTitle4 = $_POST['projectTitle4'];
+$projectTitleLink4 = $_POST['projectTitleLink4'];
+$projectImage4 = $_POST['projectImage4'];
+$projectId4 = $_POST['projectId4'];
+
+if ($projectTitle4 != NULL && $projectTitleLink4 != NULL && $projectImage4 != NULL && $projectId4!= NULL) {
+    $updatedInfo = updateProject($db, $projectTitle4, $projectTitleLink4, $projectImage4, $projectId4);
+    echo $updatedInfo;
+
+}
+
+$projectTitle5 = $_POST['projectTitle5'];
+$projectTitleLink5 = $_POST['projectTitleLink5'];
+$projectImage5 = $_POST['projectImage5'];
+$projectId5 = $_POST['projectId5'];
+
+if ($projectTitle5 != NULL && $projectTitleLink5 != NULL && $projectImage5 != NULL && $projectId5!= NULL) {
+    $updatedInfo = updateProject($db, $projectTitle5, $projectTitleLink5, $projectImage5, $projectId5);
+    echo $updatedInfo;
+
+}
+
+$projectTitle6 = $_POST['projectTitle6'];
+$projectTitleLink6 = $_POST['projectTitleLink6'];
+$projectImage6 = $_POST['projectImage6'];
+$projectId6 = $_POST['projectId6'];
+
+if ($projectTitle6 != NULL && $projectTitleLink6 != NULL && $projectImage6 != NULL && $projectId6!= NULL) {
+    $updatedInfo = updateProject($db, $projectTitle6, $projectTitleLink6, $projectImage6, $projectId6);
+    echo $updatedInfo;
+
+}
+
+$projectTitle7 = $_POST['projectTitle7'];
+$projectTitleLink7 = $_POST['projectTitleLink7'];
+$projectImage7 = $_POST['projectImage7'];
+$projectId7 = $_POST['projectId7'];
+
+if ($projectTitle7 != NULL && $projectTitleLink7 != NULL && $projectImage7 != NULL && $projectId7!= NULL) {
+    $updatedInfo = updateProject($db, $projectTitle7, $projectTitleLink7, $projectImage7, $projectId7);
+    echo $updatedInfo;
+
+}
+
+$projectTitle8 = $_POST['projectTitle8'];
+$projectTitleLink8 = $_POST['projectTitleLink8'];
+$projectImage8 = $_POST['projectImage8'];
+$projectId8 = $_POST['projectId8'];
+
+if ($projectTitle8 != NULL && $projectTitleLink8 != NULL && $projectImage8 != NULL && $projectId8!= NULL) {
+    $updatedInfo = updateProject($db, $projectTitle8, $projectTitleLink8, $projectImage8, $projectId8);
+    echo $updatedInfo;
+
+}
+
+$projectTitle9 = $_POST['projectTitle9'];
+$projectTitleLink9 = $_POST['projectTitleLink9'];
+$projectImage9 = $_POST['projectImage9'];
+$projectId9 = $_POST['projectId9'];
+
+if ($projectTitle9 != NULL && $projectTitleLink9 != NULL && $projectImage9 != NULL && $projectId9!= NULL) {
+    $updatedInfo = updateProject($db, $projectTitle9, $projectTitleLink9, $projectImage9, $projectId9);
+    echo $updatedInfo;
+
+}
+
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Portfolio</title>
+</head>
+<body>
+
+<h1>Portfolio</h1>
+
+    <div>
+        <h2>Current Values</h2>
```

Comment:
use a function that creates these for you and does it with a loop to save repeated code

Reply:
same for all in this file